client/web/src/components/CommitSearchResultMatch.tsx

```diff
@@ -151,7 +152,7 @@ export class CommitSearchResultMatch extends React.Component<
                 partialVisibility={true}
                 offset={this.visibilitySensorOffset}
             >
-                <div className="commit-search-result-match">
+                <div className={styles.commitSearchResultMatch}>
                     {this.props.item.repoLastFetched &&
                         LastSyncedIcon({ lastSyncedTime: this.props.item.repoLastFetched })}
```

Comment:
This should also be used like a React component: 
```suggestion
                        <LastSyncedIcon lastSyncedTime={this.props.item.repoLastFetched} />}
```

Reply:
Whoops. Thanks for the catch.